Sources/ArcGISToolkit/Components/Offline/Utilities/URL+Offline.swift

```diff
@@ -16,11 +16,15 @@ import ArcGIS
 import Foundation
 
 extension URL {
+    static func offlineManagerDirectory() -> URL {
```

Comment:
```suggestion
    /// The path to the offline manager directory.
    /// `Documents/com.esri.ArcGISToolkit.offlineManager/`
    /// - Returns: A URL to the offline manager directory.
    static func offlineManagerDirectory() -> URL {
```
Suggested documentation.